src/Meanbee/Magedbm/Command/SelfUpdateCommand.php

```diff
@@ -0,0 +1,96 @@
+<?php
+namespace Meanbee\Magedbm\Command;
+
+use Herrera\Json\Exception\FileException;
+use Herrera\Phar\Update\Manager;
+use Herrera\Phar\Update\Manifest;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class SelfUpdateCommand extends BaseCommand
+{
+
+    const MANIFEST_FILE = 'https://s3-eu-west-1.amazonaws.com/magedbm-releases/manifest.json';
+
+    /**
+     * Configure the command parameters.
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $this
+            ->setName('phar:update')
+            ->setAliases(array('selfupdate', 'self-update'))
+            ->addArgument(
+                'version',
+                InputArgument::OPTIONAL,
+                'Updates to version-number (i.e. 1.3.2). When omitted will update to the latest version'
+            )
+            ->addOption('major', 'm', InputOption::VALUE_NONE, 'Lock to current major version')
```

Comment:
Looking at how this is used, seems the effect is opposite from the description. Specifying a `-m` would upgrade from 1.x to 2.x while leaving it out would not.